pkg/test/framework/components/apps/local/envoy/admin_util.go

```diff
@@ -167,7 +170,10 @@ func IsClusterPresent(cfg *envoy_admin_v2alpha.ConfigDump, clusterName string) b
 // IsOutboundListenerPresent inspects the given Envoy config dump, looking for the given listener.
 func IsOutboundListenerPresent(cfg *envoy_admin_v2alpha.ConfigDump, listenerName string) bool {
 	listeners := envoy_admin_v2alpha.ListenersConfigDump{}
-	if err := listeners.Unmarshal(cfg.Configs["listeners"].Value); err != nil {
+	if len(cfg.Configs) < 2 {
+		return false
+	}
+	if err := listeners.Unmarshal(cfg.Configs[1].Value); err != nil {
```

Comment:
Missed this originally but these are the wrong way round 🙂

Reply:
Ah, I think you looked at an old snapshot, could you check the latest change? Thank you.